plugins/org.jboss.tools.vpe/src/org/jboss/tools/vpe/editor/VpeEditorPart.java

```diff
@@ -1136,10 +1136,13 @@ public void updatePartAccordingToPreferences() {
 					.getString(IVpePreferencesPage.SELECTION_HIDDEN_BORDER_COLOR));
 			visualEditor.getXulRunnerEditor().setHiddenSelectedElementColor(VpeStyleUtil.rgbToString(rgb));
 			
-//			if (presfShowSelectionBar != selectionBar.isVisible()) {
-//				selectionBar.setRealBarVisible(presfShowSelectionBar);
-//				doVisualRefresh = true;
-//			}
+			
```

Comment:
This code is additonal edit for JBIDE-22740 - Fix updating selection bar also for VPE Editor